extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -138,6 +150,41 @@ export class IframeTransport {
     return frameData;
   }
 
+  createLongTaskObserver_() {
+    if (this.longTaskObserver_ ||
+        typeof this.ampWin_.PerformanceObserver == 'undefined') {
+      return;
+    }
+    this.longTaskObserver_ = new this.ampWin_.PerformanceObserver(entryList => {
+      if (!entryList) {
+        return;
+      }
+      entryList.getEntries().forEach(entry => {
+        if (entry && entry['entryType'] == 'longtask' &&
+            (entry['name'] == 'cross-origin-descendant' ||
+            entry['name'] == 'multiple-contexts' ||
+            (getMode().localDev &&
+            entry['name'] == 'same-origin-descendant')) &&
+            entry.attribution) {
+          entry.attribution.forEach(attrib => {
+            if (this.frameUrl_ == attrib.containerSrc &&
+                ++this.numLongTasks_ >= LONG_TASK_REPORTING_THRESHOLD_) {
+              user().warn(TAG_,
+                  'Long Task: ' +
+                  `Attribution: "${attrib.name}" ` +
+                  `Vendor: ${this.type_} ` +
+                  `Src: "${attrib.containerSrc}" ` +
+                  `Duration: ${entry.duration}ms ` +
+                  `StartTime: ${attrib.startTime} ` +
+                  `Occurrences: ${this.numLongTasks_}`);
```

Comment:
More than happy to change log message format if others have opinions.

Reply:
Should also inform the vendor frame that it is being non-performant. I would prefer separate PR for that but am flexible.